.gitlab-ci.yml

```diff
@@ -741,25 +735,28 @@ deploy-kubernetes-alerting-rules:
     - echo "deploying prometheus alerting rules"
     - kubectl -n ${NAMESPACE} patch prometheusrule ${PROMETHEUSRULE}
         --type=merge --patch "$(sed 's/^/  /;1s/^/spec:\n/' ${RULES})"
-  only:
-    refs:
-      - master
-    changes:
-      - .gitlab-ci.yml
-      - .maintain/monitoring/
-
-
+  rules:
+    - if: $CI_COMMIT_REF_NAME == "master"
+      changes:
+        - .gitlab-ci.yml
+        - .maintain/monitoring/
```

Comment:
this would have to be `.maintain/monitoring/*` actually according to https://docs.gitlab.com/ee/ci/yaml/#onlychangesexceptchanges . i was actually sure it was working before.

Reply:
fixed with `maintain/monitoring/**/*` as it has dirs inside